chain/chain/src/state_snapshot_actor.rs

```diff
@@ -144,3 +151,16 @@ pub fn get_make_snapshot_callback(
         );
     })
 }
+
+/// Sends a request to delete a state snapshot.
+pub fn get_delete_snapshot_callback(
+    state_snapshot_addr: Arc<actix::Addr<StateSnapshotActor>>,
+) -> DeleteSnapshotCallback {
+    Arc::new(move || {
+        tracing::info!(
+            target: "state_snapshot",
+            "start_delete_snapshot_callback sends `DeleteSnapshotRequest` to state_snapshot_addr");
+        state_snapshot_addr
```

Comment:
We don't need to call `flat_storage_manager.set_flat_state_updates_mode(false)` here